pkg/sdn/plugin/eventqueue.go

```diff
@@ -0,0 +1,140 @@
+package plugin
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/client/cache"
+)
+
+func NewEventQueue(keyFunc cache.KeyFunc) *cache.DeltaFIFO {
```

Comment:
Currently it's just pkg/router/controller/factory/factory.go and that's apparently going to stop using it.  @smarterclayton should I just replace pkg/client/cache/eventqueue.go with this new DeltaFIFO-based event queue, and convert everything over to use it?